core/templates/pages/error-pages/error-page-root.component.ts

```diff
@@ -18,20 +18,24 @@
 
 // This error page is used for status codes 400, 401 and 500.
 
-import { Component } from '@angular/core';
+import { Component, OnDestroy } from '@angular/core';
```

Comment:
Separate this from the other imports. Ditto elsewhere.

Reply:
Done! (I've noticed that the `downgradeComponent` import was usually found next to this import, i.e. the imports from `@angular/core` and the `downgradeComponent` import were always together, separate from the rest of the imports. I've kept this in mind while re-ordering the imports. Please let me know if this isn't right.)